openhands/integrations/github/github_service.py

```diff
@@ -695,6 +695,104 @@ async def get_microagent_content(
         # Parse the content to extract triggers from frontmatter
         return self._parse_microagent_content(file_content, file_path)
 
+    async def get_issue_comments(
```

Comment:
@openhands write a method that gets the comments from a review thread in a PR given the pr_number, thread_id, and repo_name

then use the $GITHUB_TOKEN in your env to test that this method works as expected. Use the github api to curl for a review thread and use those same params to call this method